vllm/engine/arg_utils.py

```diff
@@ -371,7 +371,10 @@ def add_cli_args(parser: FlexibleArgumentParser) -> FlexibleArgumentParser:
                             'capping to sliding window size')
         parser.add_argument('--use-v2-block-manager',
                             action='store_true',
-                            help='Use BlockSpaceMangerV2.')
+                            help='[DEPRECATED] block manager v1 has been '
+                            'removed and block manager v2 is now the default '
+                            'block manager. Setting this flag to True or False'
+                            ' has no effect on vLLM behavior.')
```

Comment:
Another option we could consider is adding a deprecate warning without removing v1 in a separate PR. And we release that PR first, followed by this PR that completely removes v1 along with this flag. Would like to hear others' thoughts.

Reply:
Agree, let's see how people react towards making block manager v2 default, and then fully remove block manager v1 after people are OK with block manager v2 as default.